datasetinsights/stats/visualization/coco_data_visualization.py

```diff
@@ -0,0 +1,78 @@
+import random
+from typing import List, Union
+
+import matplotlib.pyplot as plt
+from pycocotools.coco import COCO
+
+from datasetinsights.stats.coco_statistics import (
+    load_annotations_for_single_img,
+    load_image_from_img_ann,
+    load_img_ann_for_single_image,
+)
+
+
+def display_single_img(
```

Comment:
@Saurav-D Pleases decouple COCO api from this to allow other datasets in different format to reuse this method. I think the signature should be:

```
def display_single_image(image: str, annotation: Optional[list] = None):
    """
    Args:
        image (str): path the the image file
        annotation (list): list of 2d bounding boxes in this image
    """
    pass


def display_multiple_images(images: list, annotations: list, n_cols=3):
    """
    Args:
       images (list): list of path (string) to images files
       annotations (list): list of annotations for each image
    """
    pass

```

Reply:
Let's decouple data loading from COCO API from display images with bounding boxes so that it's open to extension for new dataset formats.